app/views/confirm-unsubscribe.html

```diff
@@ -16,7 +16,7 @@ <h1 class="heading-xlarge">
         Manage your subscriptions
       </h1>
 
-      <p>You are unsubscribing to {{ title }}.</p>
+      <p>You are unsubscribing to '{{ title }}'.</p>
```

Comment:
Do we say "You are unsubscribing to" or "You are unsubscribing from" ?

Reply:
Ah - that's not something I changed, but you're quite right, "unsubscribing to" sounds wrong.